backend/prisma/generated/helpers/index.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+var __spreadArray =
+  (this && this.__spreadArray) ||
+  function (to, from, pack) {
+    if (pack || arguments.length === 2)
+      for (var i = 0, l = from.length, ar; i < l; i++) {
+        if (ar || !(i in from)) {
+          if (!ar) ar = Array.prototype.slice.call(from, 0, i);
+          ar[i] = from[i];
+        }
+      }
+    return to.concat(ar || Array.prototype.slice.call(from));
+  };
+exports.__esModule = true;
+exports.getEnumValues = void 0;
```

Comment:
why exports here, not at the bottom like the other one?

Reply:
as the path suggests, this file is automatically generated, wont change it

Reply:
my bad these file are not needed anymore, i forgot to delete these after i tried another prisma helper for swagger